Bottom: 983e2668c68c292796b5ed74b1f4a5fa36cfff8b
Top:    e7d341ee06bacd1c15e02dcf8aa2308b5b4863de
Author: scott snyder <sss@karma>
Date:   2020-02-03 09:57:15 -0500

cmake LibDeps workaround


---

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5c89f06..ded978d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -93,5 +93,6 @@ INSTALL( PROGRAMS "./bin/aida-config" "./bin/aida-setup.sh" DESTINATION bin )
 DISPLAY_STD_VARIABLES()
 
 # generate and install following configuration files
-GENERATE_PACKAGE_CONFIGURATION_FILES( RAIDAConfig.cmake RAIDAConfigVersion.cmake AIDAConfig.cmake AIDAConfigVersion.cmake RAIDALibDeps.cmake )
+GENERATE_PACKAGE_CONFIGURATION_FILES( RAIDAConfig.cmake RAIDAConfigVersion.cmake AIDAConfig.cmake AIDAConfigVersion.cmake  )
 
+INSTALL (FILES ${PROJECT_SOURCE_DIR}/cmake/RAIDALibDeps.cmake DESTINATION lib/cmake )
diff --git a/cmake/RAIDALibDeps.cmake b/cmake/RAIDALibDeps.cmake
new file mode 100644
index 0000000..e69de29
